homeassistant/components/openweathermap/air_quality.py

```diff
@@ -0,0 +1,124 @@
+"""Support for the OpenWeatherMap (OWM) Air Pollution service."""
```

Comment:
Air quality is a legacy integration that we don't accept new platforms for. Please add a sensor entity instead.